src/pmod_maxsonar.erl

```diff
@@ -1,3 +1,11 @@
+%% -----------------------------------------------------------------------------
+%% @doc
+%% <a href="https://store.digilentinc.com/pmodmaxsonar-maxbotix-ultrasonic-range-finder/">
+%% Pmod MAXSONAR
+%% </a>
+%% module that provides data request through the UART interface.
```

Comment:
Describe that the MAXsonar cyclically sends measurements via the UART interface.  Then in the get call one gets the latest received value